common/beerocks/bwl/CMakeLists.txt

```diff
@@ -59,7 +59,7 @@ if(BWL_TYPE STREQUAL "DWPAL")
     )
 
     find_package(dwpal REQUIRED)
-    list(APPEND BWL_LIBS dwpal)
+    list(APPEND BWL_LIBS dwpal nl-3 nl-genl-3)
```

Comment:
Perhaps it's better to create a FindNL cmake file and use the standard cmake `find_package()`.
Here's an example: https://github.com/nasa/channel-emulator/blob/master/cmake/Modules/FindLibNL.cmake

Reply:
> Here's an example: https://github.com/nasa/channel-emulator/blob/master/cmake/Modules/FindLibNL.cmake

Careful with that! The project is GPL-2.0 so presumably that file is GPL-2.0 as well. As such it's OK if we include GPL-2.0 files that are used during the build (it just means that if you distribute the source code, you need to give the recipient the right to redistribute that particular file further). But it's making our license more complicated. If you do take over that file, it should get a copyright header including authorship information of the original (which is difficult to retrieve since they probably also copied it from somewhere else).

Reply:
That was just an example. We should write our own cmake find file for libnl..

Reply:
As agreed, and since its already exists for other libraries, lets do this in a separate PR as part #665.
Keeping the comment open for that.